### PR TITLE
Update commit status notification guide

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: docs/guides/notifications.md [hunk 1]
```diff
@@ -105,23 +105,40 @@ When the verbosity is set to `info`, the controller will alert if:
...
-![gitlab commit status](../_files/gitlab-commit-status.png)
-
-Currently the provider will only work with Alerts for Kustomization resources as the events have to be linked with a
-specific git commit. Any other event that does not contain a commit reference will be ignored by the provider.
-Each status will contain some additional information from the event which includes the resource kind, name and reason for the event.
-It will be displayed in the format of `{{ .Kind }}/{{ .Name }} - {{ .Reason }}`.
+The GitHub, GitLab, Bitbucket, and Azure DevOps providers are slightly different to the other providers. Instead of
+a stateless stream of events, the git notification providers will link the event with accompanying git commit which
+triggered the event. The linking is done by updating the commit status of a specific commit.
+  - [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks)
```
Comment (stefanprodan) [inline docs/guides/notifications.md:111]: I think we need a new line before this for mkdocs to render the list

**Review by blaquezzz — APPROVED** [review]

**Comment by phillebaba** [comment]
@stefanprodan guide should be done now. I added screenshots for both GitHub and GitLab, as they are the most relevant. Hopefully everything will render properly.

File: docs/guides/notifications.md [hunk 2]
```diff
@@ -156,22 +213,148 @@ metadata:
...
+| ![commit status GitHub successful](../_files/commit-status-github-success.png) | ![commit status GitLab successful](../_files/commit-status-gitlab-success.png) |
+
+Generate error
+
+A deployment failure can be forced by setting an invalid image tag in the podinfo deployment.
 ```yaml
-apiVersion: v1
-kind: Secret
+apiVersion: apps/v1
+kind: Deployment
```
Comment (stefanprodan) [inline docs/guides/notifications.md:257]: Please make this into:

```yaml
apiVersion: apps/v1
kind: Deployment
spec:
  template:
    spec:
      containers:
      - name: podinfod
        image: ghcr.io/stefanprodan/podinfo:fake
```

**Comment by stefanprodan** [comment]
We need to change this:
```
Copy the manifest content in the "kustomize" directory into the directory "staging-cluster/flux-system/podinfo"
```

To:
```
Copy the manifest content in the "kustomize" directory into the directory "./clusters/my-cluster/podinfo"
```

And update the path to `./clusters/my-cluster` in this doc to match the get started and install docs.

**Comment by phillebaba** [comment]
Ah the get started guide has changed with new paths. I was running of some older repo.

**Review by stefanprodan — APPROVED** [review]
LGTM 

Thanks @phillebaba 🥇

**Pull request merged**